DependencyInjection/HWIOAuthExtension.php

```diff
@@ -135,6 +135,7 @@ public function createResourceOwnerService(ContainerBuilder $container, $name, a
 
         $definition->replaceArgument(2, $options);
         $definition->replaceArgument(3, $name);
+        $definition->setPrivate(false);
```

Comment:
It should be probably:
```
$definition->setPublic(true);

Reply:
Crap, why do they made 2 similar methods. Will try.

Reply:
Yep, surprisingly setPublic works better
Thanks